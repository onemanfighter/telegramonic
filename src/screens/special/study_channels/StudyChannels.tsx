const StudyChannels = () => {
  return (
    <div>
      <h1>Study Channels</h1>
    </div>
  );
};

export default StudyChannels;
