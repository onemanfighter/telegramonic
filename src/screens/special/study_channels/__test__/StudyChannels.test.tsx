import { render } from '@testing-library/react';
import StudyChannels from '../StudyChannels';

describe('StudyChannels screen', () => {
  it('should render correctly', () => {
    const { container } = render(<StudyChannels />);

    expect(container).toMatchSnapshot();
  });
});
