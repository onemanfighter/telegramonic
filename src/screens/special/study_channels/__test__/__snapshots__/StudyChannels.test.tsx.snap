// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StudyChannels screen should render correctly 1`] = `
<div>
  <div>
    <h1>
      Study Channels
    </h1>
  </div>
</div>
`;
